libraries/botbuilder-runtime-integration-restify/src/index.ts

```diff
@@ -30,32 +37,38 @@ const defaultOptions: Options = {
 export async function start(
     applicationRoot: string,
     settingsDirectory: string,
-    options = defaultOptions
+    options: Partial<Options> = {}
 ): Promise<void> {
     const [services, configuration] = await getRuntimeServices(applicationRoot, settingsDirectory);
-    const server = await makeServer(services, configuration);
+    const { port, messagingEndpointPath } = tests.isObject(options)
```

Comment:
I don't think we need the type tests anymore - `Object.assign({}, defaultOptions, options)` should yield a non-partial options object, as `options` is of `Partial<Options>` type and is defaulted to `{}`.

See [Typescript playground](https://www.typescriptlang.org/play?ssl=1&ssc=1&pln=13&pc=2#code/C4TwDgpgBA8mwEsD2A7AzlAvFA3gWACgpioAzJJALijWACcEUBzAbkJKgCMBDO62hszYEAvoUIBjVLSgATCKW4BXADbA4iadQ3J0WXOxLkqUAETHTAGkPEefM3auExBQqSUoJmlFADuCYAALHWkACjBeRG4VEPRqAAVIhGiAHli0AD4ASm14XQx8IhI6CGAlOh8YTgArCC8AOm40NAQmFFCcEUs5BWU1dO6IuiiYvOks4REgA).